canvas_cli/main.py

```diff
@@ -1,12 +1,19 @@
 import importlib.metadata
+import json
+import random
 from pathlib import Path
 from typing import Optional
 
+import grpc
 import typer
+from typing_extensions import Annotated
 
 from canvas_cli.apps import plugin
 from canvas_cli.apps.logs import logs as logs_command
 from canvas_cli.utils.context import context
+from generated.messages.events_pb2 import Event as PluginRunnerEvent
```

Comment:
some linter should grumble about this, imports from the same file should be grouped

Reply:
Strangely, isort is causing this. I group them, isort separates.

<img width="651" alt="Screenshot 2024-06-17 at 3 13 37 PM" src="https://github.com/canvas-medical/canvas-plugins/assets/636504/f749c503-e8bf-40a1-82e7-ca953174dab4">

Reply:
oh, well, I will not argue with `isort`! ignore me haha